src/test/javascript/spec/pipe/artemis-date.pipe.spec.ts

```diff
@@ -30,35 +30,47 @@ describe('ArtemisDatePipe', () => {
         pipe = new ArtemisDatePipe(translateService);
     });
 
+    it('Return empty string if given date time is null', () => {
+        let localizedDateTime = pipe.transform(null);
+        expect(localizedDateTime).to.be.equal('');
+        localizedDateTime = pipe.transform(moment(null));
+        expect(localizedDateTime).to.be.equal('');
+    });
+
+    it('Return empty string if given date time is invalid moment object', () => {
+        const localizedDateTime = pipe.transform(moment('2019-02-333'));
+        expect(localizedDateTime).to.be.equal('');
+    });
+
     describe('en locale', () => {
         beforeEach(() => {
             dateTime.locale('en');
             translateService.currentLang = 'en';
         });
 
         describe('without seconds', () => {
-            it('Should return format equal to "14/4/20 9:27" with format parameter set to "short"', () => {
+            it('Should return format equal to "2020-04-14 09:27" with format parameter set to "short"', () => {
                 const localizedDateTime = pipe.transform(dateTime, 'short');
-                const format = 'D/M/YY H:m';
+                const format = 'YYYY-MM-DD HH:mm';
                 expect(ArtemisDatePipe.format('en', 'short')).to.be.equal(format);
                 expect(localizedDateTime).to.be.equal(dateTime.format(format));
-                expect(localizedDateTime).to.be.equal('14/4/20 9:27');
+                expect(localizedDateTime).to.be.equal('2020-04-14 09:27');
             });
 
-            it('Should return format equal to "Apr 14, 2020 9:27 AM" with format parameter set to "long"', () => {
+            it('Should return format equal to "Apr 14, 2020 09:27 AM" with format parameter set to "long"', () => {
```

Comment:
```suggestion
            it('Should return format equal to "Apr 14, 2020 09:27" with format parameter set to "long"', () => {
```
should fit the actual value